base/compiler/abstractinterpretation.jl

```diff
@@ -2022,6 +2191,10 @@ function typeinf_local(interp::AbstractInterpreter, frame::InferenceState)
                     lhs = stmt.args[1]
                     if isa(lhs, SlotNumber)
                         changes = StateUpdate(lhs, VarState(t, false), changes, false)
+                    elseif isa(lhs, GlobalRef)
+                        tristate_merge!(frame, Effects(ALWAYS_TRUE, ALWAYS_FALSE, TRISTATE_UNKNOWN, ALWAYS_TRUE))
```

Comment:
style nit: could you add comments to each parameter:
```suggestion
                        tristate_merge!(frame, Effects(#=consistent=#ALWAYS_TRUE, #=effect_free=#ALWAYS_FALSE, #=nothrow=#TRISTATE_UNKNOWN, #=terminates=#ALWAYS_TRUE))
```